scripts/scil_tractogram_assign_uniform_color.py

```diff
@@ -134,6 +147,20 @@ def main():
             for key in dict_colors.keys():
                 if key in base:
                     color = dict_colors[key]
+
+            if color is None and args.overwrite:
+                updated_dict = True
+                logging.warning('No color found for {} in your '
+                                'dict_colors. Generating a new color.'
+                                .format(base))
+                # Generate a new color that is not in the dict_colors
+                # and not already used.
+                new_color = distinguishable_colormap(nb_colors=1,
+                                                     exclude=rgb_colors)
+                new_color = [tc*255 for tc in new_color]
+                color = '0x%02x%02x%02x' % tuple(new_color[0].astype(int))
+                dict_colors[base] = color
+            
             if color is None:
```

Comment:
Maybe say here that adding -f would solve their problem and save a new dictionnary?